cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
Yes, they may be reusable across different distros. This PR only addresses the stuff we have already checked in. When I step on the master side work, I will be more clear about what master configuration should be reused. In addition, this refactoring is supposed to handle the trusty node config. Step-by-step work is easier to track and merge.